token/lib/token.js

```diff
@@ -124,14 +127,14 @@ class Token {
     if (alreadyPaused) {
       throw new Error('Token is already paused')
     }
-    const tokenOwner = await contract.methods.owner().call()
-    if (tokenOwner.toLowerCase() != sender.toLowerCase()) {
-      throw new Error(`Sender ${sender} is not owner of token contract (${tokenOwner})`)
-    }
+    await this.ensureContractOwner(networkId, sender)
 
     const transaction = contract.methods.pause()
     await this.sendTransaction(networkId, transaction, { from: sender })
-    if (await contract.methods.paused().call() !== true) {
+    if (
+      !this.config.multisig &&
+      await contract.methods.paused().call() !== true
```

Comment:
OK, I've generalized it and put it in `src/utils/retries.js`. I've left the indexer code alone, because changing it would require testing and making that code uglier.